proxy/src/coco/git_helper.rs

```diff
@@ -0,0 +1,46 @@
+//! git-remote-rad git helper related functionality.
+
+use crate::config;
+
+use crate::error;
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+
+/// Filename of the git helper binary.
+pub const HELPER_BINARY_NAME: &str = "git-remote-rad";
```

Comment:
Nit: I'd call it `GIT_REMOTE_RAD`. The name here is too generic.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/769/commits/f9155636d1ef080844cb06df7c1ee1c3fbe4903f